hamilton/plugins/polars_extensions.py

```diff
@@ -15,6 +15,7 @@
     Tuple,
     Type,
     Union,
+    Literal,
```

Comment:
Any suggestion about this, as it is failing test cases.

Reply:
yep we need to gate this:
```python
try:
    from typing import Literal
except ImportError:
    from typing_extensions import Literal
```